runtime/doc/syntax.txt

```diff
@@ -4855,6 +4855,8 @@ IncSearch	'incsearch' highlighting; also used for the text replaced with
 							*hl-LineNr*
 LineNr		Line number for ":number" and ":#" commands, and when 'number'
 		or 'relativenumber' option is set.
+                                                        *hl-IncSubstitute*
```

Comment:
There's a secret and really smart reason, but if I told you, I'd have to kill you.

So I'm just gonna fix that :D